tests/Feature/ApiAuthTest.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Tests\Feature;
 
-use Illuminate\Foundation\Testing\DatabaseMigrations;
```

Comment:
Is this not required?

Reply:
Nope, it was an unused import.
The class just need/use RefreshDatabase.